services/flathub/flathub.service.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { InvalidResponse, NotFound } = require('..')
+const { renderVersionBadge } = require('../version')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  currentReleaseVersion: Joi.string().required(),
```

Comment:
I'm not familiar with Flathub, so just want to confirm that we need the schema to be this relaxed? Can versions be basically anything?

Reply:
Confirmed below that version values are not constrained